src/java_bytecode/Makefile

```diff
@@ -21,6 +21,7 @@ SRC = bytecode_info.cpp \
       java_root_class.cpp \
       java_types.cpp \
       java_utils.cpp \
+      java_parse_options.cpp \
```

Comment:
Should be in alphabetical order (this definitely needs to be added to the githook and tested automatically)